librarian_background/queues.py

```diff
@@ -219,6 +232,18 @@ def consume_queue_item(session_maker: Callable[[], "Session"]) -> bool:
             # Nothing to do!
             return False
 
+        # Check if the librarian is enabled.
+        librarian = (
+            session.query(Librarian)
+            .filter_by(name=queue_item.destination)
+            .one_or_none()
+        )
+
+        if librarian is None or not librarian.transfers_enabled:
+            # We can't do anything with this librarian, but there may be other
+            # librarians that are enabled.
+            return True
```

Comment:
Why do you return `True` here?

There is no librarian or it is disabled, so nothing got consumed. I think this needs to be `False`

Reply:
Returning False stops the task and skips all following items in the list (this is used specifically for the case when we have too many globus tasks in the queue). There could be two librarians connected to this one, one of which is enabled and one of which is disabled.